modules/libcom/test/epicsEnvTest.c

```diff
@@ -72,8 +70,43 @@ MAIN(epicsEnvTest)
     value = getenv(PARENT);
     testOk(value && (strcmp(value, PARENT) == 0),
             "PARENT environment variable not modified");
+}
+
+static void testChangeEnv(void)
+{
+    const char *foo = "foo", *bar = "bar", *name = "testChangeEnv";
```

Comment:
Why does the code below sometimes use `name` and sometimes a literal `"testChangeEnv"`? It's a little confusing to read, and up to the compiler whether all the literals would return the same pointer.

Reply:
A (perhaps futile) attempt to detect if `epicsSetEnv()` store, and later clobbers, the `const char*` passed to it.